opentelemetry-api/src/global/trace.rs

```diff
@@ -426,21 +413,3 @@ pub fn shutdown_tracer_provider() {
         GlobalTracerProvider::new(NoopTracerProvider::new()),
     );
 }
-
-/// Force flush all remaining spans in span processors.
-///
-/// Use the [`global::handle_error`] to handle errors happened during force flush.
-///
-/// [`global::handle_error`]: crate::global::handle_error
-pub fn force_flush_tracer_provider() {
```

Comment:
I think it maybe tricky to force push the sdk tracer provide after we removed it from APIs. Mostly because we need to do a type conversion from `dyn ObjectSafeTracerProvider + Send + Sync` to sdk tracer provider. Maybe we can provide a util function in sdk to help with the type conversion? Something like

```
pub fn force_push(provider GlobalTracerProvider) {
        // convert to sdk tracer provider, if it is not sdk provider then this function is noop
        // force_push
}
```

Reply:
Hm this api seems relatively rarely used. Maybe it's sufficient for users to be able to use the `TracerProvider::builder()` api instead of the pipeline installs if they want to retain the ability to force flush?

Reply:
Make sense. We probably need to add some examples to demonstrate how to do that but can do it on a follow-up PR.

Reply:
Hi @TommyCpp , could you please point me to the example if there is any? I used the force_flush from global and now I'm not sure how to do it.. thanks!